marimo/_messaging/ops.py

```diff
@@ -127,8 +129,25 @@ class CellOp(Op):
     console: Optional[Union[CellOutput, List[CellOutput]]] = None
     status: Optional[RuntimeStateType] = None
     stale_inputs: Optional[bool] = None
+    run_id: Optional[RunId_t] = None
     timestamp: float = field(default_factory=lambda: time.time())
 
+    def __post_init__(self) -> None:
+        try:
+            # run_id context is only set during a run
+            if self.status is not None:
+                self.run_id = RUN_ID_CTX.get()
+        except LookupError:
+            # Be specific about the exception we're catching
+            # The context variable hasn't been set yet
+            # TODO: where are these warnings coming from?
+            # good enough to silence for now?
+            LOGGER.warning("No run_id context found, setting to None")
```

Comment:
i assume this warning is still in a lot of places. we can just remove this line and the TODO if things are working as expected